app/services/environment.rb

```diff
@@ -13,7 +13,7 @@ def api_token_authenticator
     @api_token_authenticator ||= ApiTokenAuthenticator.new(
       identifier_store: api_token_identifier_store,
       public_key: OpenSSL::PKey::RSA.new(File.read(ENV.fetch('API_TOKEN_PUBLIC_KEY_FILE'))),
-      obo_token_authenticator: UpvsEnvironment.sso_support? ? obo_token_authenticator : nil,
+      obo_token_authenticator: obo_token_authenticator,
```

Comment:
V com je tento zapis lepsi?

Reply:
toto suvisi s https://github.com/slovensko-digital/slovensko-sk-api/pull/46/files#diff-d77a9cc5f4071c5e7f3420a8b4c1849bR32 kde ide o to, ze nechceme aby nam environment vratil nejaky (mozno pokazeny, mozno by padla vynimka) OBO token authenticator pokial to nedava zmysel (SSO nie je podporene)

Reply:
Ja viem, ze to suvisi len mi to prislo divne, ze namiesto jedneho miesta odkial sa to vzdy instancuje a jednej podmienky odrazu mame 2 podmienky na 2 miestach.

Reply:
chapem, no slo mi o to aby ti toto spadlo na `nil` ak by si nahodou chcel pouzit OBO token authenticator (alebo ten jeho assertion store) ak SSO nie je podporene:
```ruby
Environment.obo_token_authenticator.verify_token(...)
```